mycore-base/src/main/resources/deprecated.properties

```diff
@@ -56,6 +56,7 @@ MCR.classifications_classification_cache_size=MCR.Persistence.Classification.Sto
 MCR.classifications_category_cache_size=MCR.Persistence.Classification.Store.CacheSize.Categ
 
 MCR.googlesitemap.types=MCR.GoogleSitemap.Types
+MCR.GoogleSitemap.Types=test
 MCR.googlesitemap.freq=MCR.GoogleSitemap.Freq
```

Comment:
you have to write the new property name behind the old one
MCR.GoogleSitemap.Types=MCR.GoogleSitemap.SolrQuery